InvenTree/label/models.py

```diff
@@ -146,6 +146,37 @@ def __str__(self):
         max_length=100,
     )
 
+    multipage = models.BooleanField(
+        default=False,
+        verbose_name=_('Print multiple labels to a single page'),
+    )
+
+    page_width = models.FloatField(
+        default=210,
+        verbose_name=_('Width of one page [mm]'),
+        help_text=_("The number of label columns per page will be automatically calculated from the label's width"),
+        validators=[MinValueValidator(2)]
+    )
+
+    page_height = models.FloatField(
+        default=297,
+        verbose_name=_('Height of one page [mm]'),
+        help_text=_("The number of label rows per page will be automatically calculated from the label's height"),
+        validators=[MinValueValidator(2)]
+    )
+
+    multipage_border = models.CharField(
+        choices=[
+            ('0.5mm solid #000;', _('Solid')),
```

Comment:
isn't the `;` at the end to much, as you also have it in the css template?

Reply:
I put the border styling between if - endif so it is not even going to be rendered. I also changed the db default values.